src/main/java/cl/transbank/pos/utils/BaseResponse.java

```diff
@@ -12,6 +12,14 @@
   private transient long swigCPtr;
   protected transient boolean swigCMemOwn;
 
+  @Override
+  public String toString() {
+    return "BaseResponse{" +
+            "swigCPtr=" + swigCPtr +
+            ", swigCMemOwn=" + swigCMemOwn +
+            '}';
+  }
+
```

Comment:
Since this file is autogenerated by Swig, nex time we change the Wrapper, this change will be lost

Reply:
I think the swig generator is a different version and produced the change.

Had to regenerate to add the enums of the responses and baudrates.